src/index.js

```diff
@@ -43,10 +44,13 @@ let shallowRender = (vnode, context) => renderToString(vnode, context, SHALLOW);
 
 const EMPTY_ARR = [];
 function renderToString(vnode, context, opts) {
+	const previousSkipEffects = options[_skipEffects];
+	options[_skipEffects] = true;
```

Comment:
Do we know if Terser inlines this? I'm 99% sure it will, just we're using the minified properties everywhere else.

Reply:
Pushed a commit that inlines it.